cocos/platform/android/jni/JniImp.cpp

```diff
@@ -144,8 +147,20 @@ extern "C"
         g_app->start();
     }
 
+    JNIEXPORT void Java_org_cocos2dx_lib_Cocos2dxRenderer_nativeDone(JNIEnv*  env, jobject thiz)
+    {
+        g_scriptEngineDone = true;
+        LOGD("CocosRenderer.nativeExit");
```

Comment:
init <-> destroy. So probably, nativeDestroy should be a better name.

Reply:
create<->destroy  init<->done